app/mm-redux/selectors/entities/roles.ts

```diff
@@ -154,7 +156,7 @@ export const getMyChannelPermissions = reselect.createSelector(
                     for (const permission of roles[roleName].permissions) {
                         permissions.add(permission);
                     }
-                    roleFound = true && teamRoleFound;
+                    roleFound = true && (teamRoleFound || teamId === undefined);
```

Comment:
you should also check for empty as in `teamId === ''` as DM's or GM's do have this value set to an empty string